perl/rest.txt

```diff
@@ -0,0 +1,2 @@
+t/00_compile.t
```

Comment:
Do we need rest and subset file?

Reply:
Not really. 
Removed.
https://github.com/launchableinc/examples/pull/67/commits/6aff50d4fb5aebc3472aa90c39db453ed17dfd3a